internal/ui/main_page.go

```diff
@@ -0,0 +1,53 @@
+package ui
+
+import (
+	"github.com/rivo/tview"
+)
+
+func MainPage(app *tview.Application) error {
+	layoutListDownload := tview.NewFlex().
+		SetDirection(tview.FlexRow)
+
+	txtDownloads := tview.NewTextView().
+		SetText("Downloads").
+		SetTextAlign(tview.AlignLeft)
+
+	listDownload := tview.NewList().
+		AddItem("123456781", "Replay 1", '1', nil).
+		AddItem("123456781", "Replay 2", '2', nil)
+
+	textView := tview.NewTextView().
+		SetDynamicColors(true).
+		SetTextAlign(tview.AlignCenter).
+		SetText("DotA2 Replay")
+	textView.SetBorder(true)
+
+	form := tview.NewForm().
+		AddInputField("Match ID", "", 20, nil, nil).
+		AddButton("Find", func() {
+			app.SetFocus(listDownload)
```

Comment:
this should open a new page (unimplemented, empty page will be fine for the scope of this PR) where replay will be played later instead